cassandra/cluster.py

```diff
@@ -4206,7 +4206,7 @@ def _handle_schema_change(self, event):
         delay = self._delay_for_event_type('schema_change', self._schema_event_refresh_window)
         self._cluster.scheduler.schedule_unique(delay, self.refresh_schema, **event)
 
-    def wait_for_schema_agreement(self, connection=None, preloaded_results=None, wait_time=None):
+    def wait_for_schema_agreement(self, target_connection=None, preloaded_results=None, wait_time=None):
 
         total_timeout = wait_time if wait_time is not None else self._cluster.max_schema_agreement_wait
         if total_timeout <= 0:
```

Comment:
Isn't this a breaking change requiring a major version bump? Code that called this using keyword arguments (`control_connection.wait_for_schema_agreement(connection=something)`) will stop working.

Reply:
reverted